TicTacToeForBot.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[52]:
+
+
+import random
+
+def drawBoard(board):
+    #Эта функция рисует игровую доску с выполненными ходами
+
+     #"Доска" является списком из 10 строк которые рисуют доску в 
+#символьной графике
+    print(' | |')
+    print(' ' + board[7] + ' | ' + board[8] + ' | ' + board[9])
+    print(' | |')
+    print('---+---+---')
+    print(' | |')
+    print(' ' + board[4] + ' | ' + board[5] + ' | ' + board[6])
+    print(' | |')
+    print('---+---+---')
+    print(' | |')
+    print(' ' + board[1] + ' | ' + board[2] + ' | ' + board[3])
+    print(' | |')
+
+def inputPlayerLetter():
+     #Позволяет игроку выбрать символ, которым он хочет играть
+     #Возвращает список с буквой игрока в качестве первого элемента и 
+#буквой компьютера в качестве второго элемента
+     letter = ''
+     while not (letter == 'Х' or letter == 'О'):
+        print('Каким знаком вы будете играть? (Х или О)')
+        letter = input().upper()
+
+     #Первым элементом возвращаемого списка всегда должен быть знак игрока.
+     if letter == 'Х':
+        return ['Х', 'О']
+     else:
+        return ['О', 'Х']
+
+def whoGoesFirst():
+     #Случайно определяется, кто будет ходить первым
+    if random.randint(0, 1) == 0:
+         return 'компьютер'
+    else:
+         return 'игрок'
+
+def playAgain():
+     #Эта функция возвращает True, если игрок хочет сыграть еще раз. 
+#Иначе False.
+     print('Вы хотите сыграть еще раз? (да или нет)')
+     return input().lower().startswith('д')
+
+def makeMove(board, letter, move):
+     board[move] = letter
+
+def isWinner(bo, le):
+     #Функция учитывает позицию на доске и текщий ход игрока. Возвращает 
+#True, если игрок выиграл
+     #Мы используем bo вместо доски и le вместо полных имен переменных
+     return ((bo[7] == le and bo[8] == le and bo[9] == le) or #Верхняя линия
```

Comment:
Is it possible to write this part in a more short way?

Reply:
Done